LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
@sarina @jbarciauskas +1 to Sarina's point - this requirement from the AGPL is very central to getting most of our clients to accept contributing the changes they pay for. From past experience on projects without that requirement, if we move away from the AGPL, it will become much harder to get clients to agree, and will likely significantly diminish the amount of contributions we are able to do.